cmake/DAGMCConfig.cmake.in

```diff
@@ -0,0 +1,22 @@
+# - Config file for the Direct Accelerated Geometry Monte Carlo (DAGMC) package
+# It defines the following variables
+#  DAGMC_INCLUDE_DIRS - include directories for FooBar
```

Comment:
FooBar

Reply:
hah yup. Will fix

- [ ] update description of `DAGMC_INCLUDE_DIRS` variable